src/app/shell/details/details-tabs/reviews/reviews.component.ts

```diff
@@ -55,17 +57,22 @@ export class ReviewsComponent implements OnInit, OnDestroy {
   ngOnInit(): void {
     this.getParentData();
     this.getWorkshopRatingList();
-    this.checkIfAllowedToReview(this.parent.id)
-    
+    this.checkIfAllowedToReview(this.parent.id);
+    this.checkIfReviewed(this.parent.id);
```

Comment:
mode this to parent subscription, because at this moment probably we won't have parent id yet